lib/cli/src/commands/create_exe.rs

```diff
@@ -573,9 +621,6 @@ impl CreateExe {
             }
             cmd.arg("-lunwind");
             cmd.arg("-OReleaseSafe");
-            cmd.arg("-fstrip");
```

Comment:
Yes, they were causing issues with different zig versions. Stripping is not important, it was only saving ~1 MB on a 30MB binary since it doesn't strip the debug info from the libwasmer.a, but the debug info from the wasmer_main.o file (which is very small). So unless zig can do LTO, we're not going to get massive amounts of size reduction here.